plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button, Card } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './product-card.scss';
+import { Product } from '../product-list-content/product-list-content';
+import { getAdminSetting } from '../../../../client/utils/admin-settings';
+
+export interface ProductCardProps {
+	type?: string;
+	product: Product;
+}
+
+function ProductCard( props: ProductCardProps ): JSX.Element {
+	const { product } = props;
+	const currencySymbol = getAdminSetting( 'currency' )?.symbol;
+	return (
+		<Card className="woocommerce-marketplace__product-card">
+			<div className="woocommerce-marketplace__product-card__content">
+				<div className="woocommerce-marketplace__product-card__header">
+					<div className="woocommerce-marketplace__product-card__details">
+						{ product.icon && (
+							<div className="woocommerce-marketplace__product-card__icon">
+								<img
+									src={ product.icon }
+									alt={ product.title }
+								/>
+							</div>
```

Comment:
I get the feeling we have some unnecessary elements in the markup in this component. For example, do we really need the div here? Could we make this work with just the `img`?